### DISCUSSION TITLE
Sniffing interfaces health

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#8588] [Unsupported Versions] [answered]
hi,

i have come to the understanding that all sniffing interfaces are bond together to one bond interface.
does security onion has a mechanism to alert if one of the sniffing interfaces is down or pcap's are not picked up ?

**Comment by dougburks** [comment, marked as answer]
You can use Grafana to monitor the amount of traffic seen by the bond interface. You could also create an alarm if that traffic drops to a certain level.
https://docs.securityonion.net/en/2.3/grafana.html
https://youtu.be/8FmZ4MRe8Uk

**Reply by udi-mosh** [reply]
thank you very much
